src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -0,0 +1,1091 @@
+	}
+}
+
+interface ExtensionSuggestion {
+	name: string;
+	title?: string;
+	description?: string;
+	id: string;
+	isKeymap?: boolean;
+	isCommand?: boolean;
+	isExtensionPack?: boolean;
+	icon?: string;
+	link?: string;
+	extensionPackExtensions?: any[];
+}
+
+
+const extensionPacks: ExtensionSuggestion[] = [
+	{
+		name: localize('welcomePage.adminPack', "SQL Admin Pack"),
+		title: localize('welcomePage.showAdminPack', "SQL Admin Pack"),
+		description: localize('welcomePage.adminPackDescription', "Admin Pack for SQL Server is a collection of popular database administration extensions to help you manage SQL Server"),
+		id: 'microsoft.admin-pack',
+		extensionPackExtensions: [
+			{
+				name: 'SQL Server Agent',
+				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-agent-extension?view=sql-server-2017',
+				icon: '../../../../../extensions/admin-pack/images/extension.png'
+			},
+			{
+				name: 'SQL Server Profiler',
+				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-profiler-extension?view=sql-server-2017',
+				icon: '../../../../../extensions/admin-pack/images/extension.png'
+			},
+			{
+				name: 'SQL Server Import',
+				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-import-extension?view=sql-server-2017',
+				icon: '../../../../../extensions/admin-pack/images/extension.png'
+			},
+			{
+				name: 'SQL Server Dacpac',
+				link: 'https://docs.microsoft.com/sql/azure-data-studio/sql-server-dacpac-extension?view=sql-server-2017',
+				icon: '../../../../../extensions/admin-pack/images/extension.png'
+			}
+		],
+		isExtensionPack: true
+	},
+];
+
+const extensions: ExtensionSuggestion[] = [
+	{ name: localize('welcomePage.powershell', "Powershell"), id: 'microsoft.powershell', description: localize('welcomePage.powershellDescription', "Write and execute PowerShell scripts using Azure Data Studio's rich query editor"), icon: 'https://raw.githubusercontent.com/PowerShell/vscode-powershell/master/images/PowerShell_icon.png', link: `command:azdata.extension.open?{"id":"microsoft.powershell"}` },
+	{ name: localize('welcomePage.dataVirtualization', "Data Virtualization"), id: 'microsoft.datavirtualization', description: localize('welcomePage.dataVirtualizationDescription', "Virtualize data with SQL Server 2019 and create external tables using interactive wizards"), icon: '../../../../../extensions/admin-pack/images/extension.png', link: `command:azdata.extension.open?{"id":"microsoft.datavirtualization"}` },
+	{ name: localize('welcomePage.PostgreSQL', "PostgreSQL"), id: 'microsoft.azuredatastudio-postgresql', description: localize('welcomePage.PostgreSQLDescription', "Connect, query, and manage Postgres databases with Azure Data Studio"), icon: 'https://raw.githubusercontent.com/Microsoft/azuredatastudio-postgresql/master/images/extension-icon.png', link: `command:azdata.extension.open?{"id":"microsoft.azuredatastudio-postgresql"}` },
+];
+
+interface Strings {
```

Comment:
I'd actually suggest just getting rid of this interface entirely. It's not doing anything useful right now.